back/src/models/Product.go

```diff
@@ -12,9 +12,9 @@ type Product struct {
 	PriceWithTaxes          int            `json:"priceWithTaxes"`
 	Description             string         `json:"description"`
 	ThumbnailURL            string         `json:"thumbnailURL"`
-	OutOfStock              string         `json:"outOfStock"              gorm:"default:false"`
+	OutOfStock              bool         	 `json:"outOfStock"              gorm:"default:false"`
 	Materials               []Material     `json:"materials"               gorm:"many2many:materials_of_products"`
-	IsHighlanderOfTheMoment bool           `json:"isHighlanderOfTheMoment" gorm:"default:false"`
+	HighlanderOrder 				uint           `json:"highlanderOrder"         gorm:"default:0"`
```

Comment:
@cedricgautier is it a situation where uint can be an issue ?

Reply:
It should be fine. I don't see any cases where we'll need a negative integer.

Reply:
Ok in next sprint we will have to think about it when we will do the validators